POLICIES.md

```diff
@@ -228,8 +231,106 @@ spec:
 
 For each `container` in the pod that is being considered for admission, a [vulnerability status](https://cloud.ibm.com/apidocs/container-registry/va#imagestatusquerypath) report is retrieved for the `image` that is specified by the container.
 
-The optional `account` parameter specifies the IBM Cloud account from where exemptions are retreived for images that match the policy repository name.
+The optional `account` parameter specifies the IBM Cloud&reg; account from where exemptions are retreived for images that match the policy repository name.
```

Comment:
We've marked IBM Cloud with TM elsewhere, which is correct?

Reply:
I've used the trademarks that are used in the IBM Cloud docs: `IBM Cloud&trade; Container Registry` and `IBM Cloud&reg;`